cfme/tests/infrastructure/test_host_drift_analysis.py

```diff
@@ -67,13 +66,16 @@ def test_host_drift_analysis(request, setup_provider, provider, host, soft_asser
         def test_host_remove_creds():
             test_host.update(
                 updates={
-                    'credentials': host.Host.Credential(
+                    'credentials': host_obj.Host.Credential(
```

Comment:
REQUIRED: @mshriver, you have assigned credentials to host's instance. Could you explain why you dediced  to remove class credentials ?